engine/src/main/java/org/terasology/rendering/nui/layers/ingame/metrics/DebugOverlay.java

```diff
@@ -168,6 +168,11 @@ public Boolean get() {
     @Override
     public void update(float delta) {
         if (metricsLabel != null) {
+            if (metricsModes.isEmpty()) {
+                metricsLabel.setText("No metric modes are available!");
+                return;
+            }
+
             metricsLabel.setText(metricsModes.get(currentMode).getMetrics());
```

Comment:
👍  for "else" block.

For metricsLabel, I need to double check. But I think that check is for:
```java
metricsLabel = find("metrics", UILabel.class);
```
If _find_ fails, most probably it returns null.